AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -190,4 +198,22 @@ private extension SearchMapView {
             }
         }
     }
+    
+    var finishButton: some View {
+        Button {
+            isSearchAndFollowLinkActive = false
+            tabSelection.selection = .dashboard
+        } label: {
+            Text(Strings.SearchMapView.finishText)
+                .font(Fonts.muliHeading2.bold())
+                .padding(.trailing, 7)
+        }
+        .overlay(
+            Capsule()
+                .frame(width: 85, height: 35)
+                .foregroundColor(.accentColor)
+                .opacity(0.1)
+        )
+        .padding(.trailing, 10)
```

Comment:
The TextField which is below has a padding of 20. I would go here for 20 too, but make it please as you see it better